src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { SyPersonioJob } from '../../../types';
+
+interface CareerDetailsStructuredDataProps {
+  position: SyPersonioJob;
+}
+
+const convertScheduleToEmploymentType = (schedule: string): string => {
+  return schedule.toUpperCase().replace('-', '_');
+};
+
+const concatDescription = (position: SyPersonioJob): string => {
+  return (
+    position.short +
+    '<br><br>' +
+    position.sections
+      .map((section) => `${section.headline} <br> ${section.descriptionHtml}`)
+      .join('<br><br>')
+  );
+};
+
+export const CareerDetailsStructuredData = ({
+  position,
+}: CareerDetailsStructuredDataProps) => {
+  const employmentType = convertScheduleToEmploymentType(position.schedule);
+
+  const data = {
+    '@context': 'https://schema.org/',
+    '@type': 'JobPosting',
+    title: position.name,
+    datePosted: position.createdAt,
+    description: concatDescription(position),
+    identifier: {
+      '@type': 'PropertyValue',
+      name: 'Satellytes Digital Consulting GmbH',
+      value: '338504-41230',
+    },
+    hiringOrganization: {
+      '@type': 'Organization',
+      name: 'Satellytes Digital Consulting GmbH',
+      sameAs: 'http://www.satellytes.com',
+      logo: 'https://we-are-hiring.cdn.personio.de/logos/41230/social/6ca8f29950517ac39fcae0d6545a1fe6.png',
+    },
+
+    jobLocation: {
+      '@type': 'Place',
+      address: {
+        '@type': 'PostalAddress',
+        streetAddress: 'Sendlinger Stra\u00dfe 52',
+        addressLocality: 'M\u00fcnchen',
+        addressRegion: 'Bayern',
+        postalCode: '80331',
+        addressCountry: 'DE',
+      },
+    },
+    jobLocationType: 'TELECOMMUTE',
```

Comment:
Another property `applicantLocationRequirements` is interesting which tells the place to be when working remotely. That would be for our hiring strategy right now Germany. But I don't want to include it right now as we otherwise forget it in the code while opening this restriction to EU. So I would like to reference EU which is our bare minimum requirement for remote work due to tax issues and timezones.

https://developers.google.com/search/docs/advanced/structured-data/job-posting#applicant-location-requirements


> For jobs in which the employee may or must work remotely 100% of the time, you must use jobLocationType. The jobLocation property isn't required if applicantLocationRequirements is present.

We will provide both to accomodate for our hybrid hiring strategy which is also valid and covered in the documentation linked above.

Reply:
This property also helps to spread visibility to EU:

> If a remote job doesn't contain applicantLocationRequirements, Google shows the job to anyone in the country specified in jobLocation.

Reply:
I added `applicantLocationRequirements`